scripts/index-search.js

```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+const matter = require('gray-matter');
+
+const root = process.cwd();
+const files = fs.readdirSync(path.join(root, 'data', 'blog'));
+
+const blogPosts = files.reduce((allPosts, slug) => {
```

Comment:
I was curious why you chose to use reduce over map here.

Reply:
Because I copy-pasted this from another file 😆 Good suggestion for `map`!